### DISCUSSION TITLE
Has anyone tried RKE2 on RHEL 9 or Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by eriksjolund** [rancher/rke2#3038] [General]
__RHEL9__ and __Ubuntu 22.04__ are not listed under the text 
"_RKE2 has been tested and validated on the following operating systems, and their subsequent non-major releases:_"
at the RKE2 [installation requirements](https://docs.rke2.io/install/requirements/) web page.

Has anyone had any success running RKE2 under Ubuntu 22.04 or RHEL 9?

I found
* https://github.com/rancher/rke2/issues/2780

but it had no comments yet.

**Comment by brandond** [comment]
They should work fine. QA and Product just haven't blessed them yet. I do all my dev work on Ubuntu 22.04.
